scripts/docker/Dockerfile.jenkins

```diff
@@ -0,0 +1,107 @@
+# VERSION 0.2 -- very basic setup of an image.
+#JENKINS expects this file to be copied as Dockerfile to the root of the repo when Jenkins job is run for building
```

Comment:
Hey @davidbernick , If I make a change to the dockerfile for gatk-protected, do I have to repeat the change here?  Why can't you use the gatk-protected Dockerfile?